basic/src/test/java/org/jboss/hal/testsuite/test/runtime/SimpleWebserviceEndpointTestCase.java

```diff
@@ -159,14 +155,17 @@ public void callServiceAndGetMetrics() throws Exception {
 
         wsePage.navigateInDeploymentsMenu();
```

Comment:
I am going to split one used page to several, better do it now, than forget about it...